### ISSUE TITLE
EOS consensus Test with test-net

### ISSUE THREAD

**Issue by Bo3o2S** [EOSIO/eos#4020]
Hello, guys

i have some question

Is EOS can consensus algoritm test with EOS test-net?

i wonder if i cosist of 21 nodes and connect to each other, consensus will working?

please, recommend anything for test consensus

Thank you for reading!

**Comment by taokayan** [comment]
Yes you can make your own testnet with up to 21 producers and consensus will be working same as the mainnet. Please configure your corresponding values of ```p2p-peer-address``` and ```producer-name``` in config.ini for each node.
